Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -66,7 +66,7 @@ public final class AlertManager {
          expireAfter: TimeInterval = 24 /* hours */ * 60 /* minutes */ * 60 /* seconds */,
          bluetoothProvider: BluetoothProvider,
          analyticsServicesManager: AnalyticsServicesManager,
-         preventIssuanceBeforePlayback: Bool = false
+         preventIssuanceBeforePlayback: Bool = true
```

Comment:
:( this change entered accidentially in another PR. I was just testing the needed change.